adapter/internal/adapter/adapter.go

```diff
@@ -158,12 +158,14 @@ func Run(conf *config.Config) {
 	defer cancel()
 
 	// log config watcher
-	// TODO: (VirajSalaka) Implement a rest endpoint to apply configurations
 	watcherLogConf, _ := fsnotify.NewWatcher()
-	errC := watcherLogConf.Add(config.GetMgwHome() + "/conf/log_config.toml")
+	logConfigPath, errC := config.GetLogConfigPath()
+	if errC == nil {
+		errC = watcherLogConf.Add(logConfigPath)
+	}
 
 	if errC != nil {
-		logger.LoggerMgw.Fatal("Error reading the log configs. ", errC)
+		logger.LoggerMgw.Error("Error reading the log configs. ", errC)
```

Comment:
This error detected for dynamic updates of the configs , isn't it? otherwise failing to read the config should kill the adapter

Reply:
```
adapter_1   | 2021-06-30 05:55:33 ERRO [parser.go:101] - [config.ReadLogConfigs.func1] [-] Proceeding with default log configuration as error occured while parsing the log configuration (26, 12): unescaped control character U+000A
```
It would print something like this and proceed.

this is for dynamic update of Log configs, not configs